examples/output/2island/boundaryAreaOverlap.json

```diff
@@ -5,7 +5,7 @@
       "sketchId": "17851",
       "classId": "nearshore",
       "groupId": null,
-      "geographyId": null,
+      "geographyId": "nearshore",
```

Comment:
Every metric should now have an assigned geographyId, the default being `nearshore`